x/ethbridge/keeper/keeper.go

```diff
@@ -100,11 +100,32 @@ func (k Keeper) ProcessBurn(ctx sdk.Context, cosmosSender sdk.AccAddress, amount
 	return nil
 }
 
+// ProcessUnburn processes the revert burn of bridged coins from the given sender
+func (k Keeper) ProcessUnburn(ctx sdk.Context, cosmosSender sdk.AccAddress, amount sdk.Coins) error {
```

Comment:
@juniuszhou What happens if we are running multiple relayer instances? Will this double or quadruple refund a user?